file-browser/src/components/SearchInput.js

```diff
@@ -5,6 +5,6 @@ import React from 'react';
 export default class SearchInput extends React.Component {
     render() {
         const props = this.props;
-        return (<div style={props.style}>SearchInput</div>)
+        return (<div style={props.style}>SearchInputs</div>)
```

Comment:
@jiamingguo so how close are you to being able to contribute to main project?
(any todo left you still need to work on?)